site/src/pages/api/blocks/update.api.ts

```diff
@@ -0,0 +1,118 @@
+import { body as bodyValidator } from "express-validator/src/middlewares/validation-chain-builders";
+
+import { getDbBlock } from "../../../lib/api/blocks/db";
+import { publishBlockFromNpm } from "../../../lib/api/blocks/npm";
+import { createAuthenticatedHandler } from "../../../lib/api/handler/authenticated-handler";
+import { ExpandedBlockMetadata } from "../../../lib/blocks";
+import { shouldAllowBlockPublishing } from "../../../lib/config";
```

Comment:
Now that we have two publishing methods, we should probably rename `shouldAllowBlockPublishing` into something like or `shouldAllowBlockNpmPublishing`, `shouldAllowNpmPublishing` or `npmPublishing`. We can also replace the condition:

```
export const shouldAllowBlockPublishing =
  process.env.NEXT_PUBLIC_NPM_PUBLISHING === "1";
```

It can be `!!`, `"true"` or `envalid.cleanEnv` + `envalid.bool`

Reply:
In addition, we can create another feature flag to allow direct publishing. I would not mix up two feature flag.s

Reply:
https://github.com/blockprotocol/blockprotocol/pull/548/commits/3cad45ddfd72c25139aea7b48a00d987b36eeab7

I don't think we need a feature flag for CLI publishing. We will use that very soon